packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1697,6 +1715,11 @@ export class ContainerRuntime
 			}
 			if (this.requestHandler !== undefined) {
 				return this.requestHandler(parser, this);
+			} else {
```

Comment:
i don't think we should do this, i don't think it will behave well

Reply:
for the same reason as above, the default object may not know how to handle requests, best to force consumers to use an explicit entrypoint once they have a real entrypoint object

Reply:
i think with the change above this becomes infinite recursion if every there is not request handler.